### PR TITLE
[NFC] Bwd multi-kernel spatial bnorm solver

### PR DESCRIPTION
**Pull request by DrizztDoUrden**
Testing in process.

### CODE REVIEW

**Review by muralinr — COMMENTED** [review]
Changes look good.  Can you please update on my benchmark progress? Also, Are tidy checks done?

**Comment by DrizztDoUrden** [comment]
My bad, tidy fix didn't get pushed. I have ran my bench, it passed, started yours.

**Comment by DrizztDoUrden** [comment]
Your bench passed as well, but it has 0 cases with the affected solver.

**Review by muralinr — APPROVED** [review]
Thanks for test results update.  I approve this PR.

**Review by junliume — APPROVED** [review]

**Comment by junliume** [comment]
@DrizztDoUrden what does NFC label mean? it was used for quite a few previous and similar PRs, but not this one :) Just out of curiosity.

**Comment by DrizztDoUrden** [comment]
> Vasilii Filippov what does NFC label mean? it was used for quite a few previous and similar PRs, but not this one :) Just out of curiosity.

No Functional Changes

**Comment by atamazov** [comment]
> @DrizztDoUrden what does NFC label mean? it was used for quite a few previous and similar PRs, but not this one :) Just out of curiosity.

If someone has No F* Clue what NFC means, then that's my bad. I brought it here from llvm, https://llvm.org/docs/Lexicon.html, sorry.

**Review by muralinr — APPROVED** [review]

File: src/ocl/batchnormocl.cpp [hunk 1]
```diff
@@ -517,227 +518,19 @@ void BatchNormBackward(Handle& handle,
...
-            }
-        }
+        MIOPEN_THROW(miopenStatusInternalError,
+                     "Batchnorm spatial has already been implemented as solver-invoker pairs");
     } // END spatial
     else
     { // PER ACT
-
-        ylocalsize           = (64 >= in_cstride) ? 64 : 256;
+
```
Comment (atamazov) [inline src/ocl/batchnormocl.cpp:528]: trailing spaces

**Review by atamazov — APPROVED** [review]
LGTM!

**Comment by atamazov** [comment]
@DrizztDoUrden Please do not delete branch until a ~ week passed since merging.

**Pull request merged**